### PR TITLE
Shared components project!

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- creation of a separate project for sharing react components
- usage of shared components within efiling-demo app
- JIRA link: https://justice.gov.bc.ca/jira/browse/FLA-128

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation

## How Has This Been Tested?

Locally

## Credit

@adodge-bcg

### CODE REVIEW

File: src/frontend/efiling-demo/src/components/page/page.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  flex-direction: row;
+}
+
+.content {
+  flex-direction: column;
+  padding-top: 30px;
+  padding-bottom: 30px;
+}
+
+@media (max-width: 575.98px) {
```
Comment (akroon3r) [inline src/frontend/efiling-demo/src/components/page/page.css:22]: whats the difference in the 0.02 px?

**Comment by akroon3r** [comment]
FAIL src/storyshots.test.js
  ● Test suite failed to run

    Cannot find module 'shared-components' from 'Home.js'

    However, Jest was able to find:
    	'./Home.js'
    	'./Home.stories.js'

**Comment by akroon3r** [comment]
whoops misclicked.

File: src/frontend/efiling-demo/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+module.exports = {
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-demo/jest.config.js:1]: fix issue with babel-jest as per https://github.com/facebook/jest/issues/6229#issuecomment-511184921

Comment (akroon3r) [reply src/frontend/efiling-demo/jest.config.js:1]: :shipit:

**Comment by akroon3r** [comment]
🚢 🚢 🚢

**Pull request merged**